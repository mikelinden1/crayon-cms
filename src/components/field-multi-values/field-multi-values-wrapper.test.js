import FieldMultiValues from './field-multi-values';

it('FieldMultiValues matches snapshot', () => {
    const props = {
        name: 'test-field',
        prop: {
            type: 'photo'
        },
        allValue: ['val1', 'val2', 'val3'],
        actions: {
            deleteMultiItem: () => {},
            multiItemSortEnd: () => {}
        }
    };

    const wrapper = shallow(<FieldMultiValues {...props} />);
    expect(wrapper).toMatchSnapshot();
});