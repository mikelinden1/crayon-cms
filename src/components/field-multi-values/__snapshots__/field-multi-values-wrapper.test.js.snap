// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FieldMultiValues matches snapshot 1`] = `
<sortableList(FieldMultiValueWrapper)
  actions={
    Object {
      "deleteMultiItem": [Function],
      "multiItemSortEnd": [Function],
    }
  }
  allValue={
    Array [
      "val1",
      "val2",
      "val3",
    ]
  }
  axis="y"
  distance={0}
  getHelperDimensions={[Function]}
  helperClass="field-multi-values-row"
  hideSortableGhost={true}
  lockOffset="50%"
  lockToContainerEdges={false}
  name="test-field"
  onSortEnd={[Function]}
  pressDelay={0}
  pressThreshold={5}
  prop={
    Object {
      "type": "photo",
    }
  }
  shouldCancelStart={[Function]}
  transitionDuration={300}
  useWindowAsScrollContainer={false}
/>
`;
